src/main/kotlin/no/skatteetaten/aurora/mokey/model/ApplicationData.kt

```diff
@@ -0,0 +1,73 @@
+package no.skatteetaten.aurora.mokey.model
+
+import com.fasterxml.jackson.databind.JsonNode
+import no.skatteetaten.aurora.mokey.controller.AuroraStatus
+import no.skatteetaten.aurora.mokey.service.Endpoint
+import no.skatteetaten.aurora.mokey.service.ManagementLinks
+
+data class ApplicationData(
+        val id: String,
+        val auroraStatus: AuroraStatus,
+        val deployTag: String,
+        val name: String,
+        val namespace: String,
+        val affiliation: String?,
+        val booberDeployId: String? = null,
+        val managementPath: String? = null,
+        val pods: List<PodDetails> = emptyList(),
+        val imageDetails: ImageDetails? = null,
+        val deployDetails: DeployDetails,
+        val sprocketDone: String? = null
+)
+
+data class PodDetails(
+        val openShiftPodExcerpt: OpenShiftPodExcerpt,
+        val managementData: ManagementData
+)
+
+data class ManagementEndpointError(
+        val message: String,
+        val endpoint: Endpoint,
+        val code: String,
+        val rootCause: String? = null
+)
+
+data class ManagementData constructor(
+        val links: ManagementLinks? = null,
+        val info: JsonNode? = null,
```

Comment:
Her synes jeg at Info bør være en Result|Health|. Hvor result har 2 implementasjoner Failed og Success. Failed kan inneholde Exception f.eks.

Slik det er nå så er det ikke noe sammenheng rundt at info er tom og at det er en feilmelding for info